content/general/concepts/json/json.md

```diff
@@ -14,9 +14,11 @@ CatalogContent:
 
```

Comment:
Could we add the following to the meta-`Tags` above?

- 'Properties'
- 'Values'